runtime/src/main/java/com/xatkit/plugins/moodle/platform/utils/MessageObject.java

```diff
@@ -59,6 +65,14 @@ public void setMessage(String message) {
     public void setUserId(String userId) {
         this.userId = userId;
     }
+    
+    /**
+     * Sets the session data of the user who sent the message
+	 * @param currentCourseAccess the sessionData to set
+	 */
+	public void setCurrentCourseAccessID(String currentCourseAccess) {
```

Comment:
If we change the parameter name, this would be a better name: `setCurrentCourseId`